astro_2.0/features/Comments/components/CommentActions/CommentActions.tsx

```diff
@@ -42,38 +39,37 @@ export const CommentActions: FC<Props> = ({
   }
 
   return (
-    <GenericDropdown
-      isOpen={open}
-      onOpenUpdate={setOpen}
-      parent={
-        <div className={styles.root}>
-          <IconButton icon="buttonMore" className={styles.rootIcon} />
-        </div>
-      }
-    >
-      <ul className={styles.menu}>
-        <li className={styles.menuItem}>
-          <Button
-            variant="transparent"
-            className={cn(styles.buttonContent)}
-            size="small"
-            onClick={onEdit}
-          >
-            <Icon name="buttonEdit" className={styles.icon} /> <span>Edit</span>
-          </Button>
-        </li>
-        <li className={cn(styles.menuItem, styles.red)}>
-          <Button
-            variant="transparent"
-            className={cn(styles.buttonContent, styles.red)}
-            size="small"
-            onClick={handleDelete}
-          >
-            <Icon name="buttonDelete" className={styles.icon} />{' '}
-            <span>Remove</span>
-          </Button>
-        </li>
-      </ul>
-    </GenericDropdown>
+    <div className={styles.root}>
+      <Tooltip
+        popupClassName={styles.tooltipPopup}
+        className={styles.tooltip}
+        placement="left"
+        overlay={<span className={styles.tooltipOverlay}>Delete</span>}
+      >
+        <Button
+          variant="transparent"
+          className={styles.button}
+          size="small"
+          onClick={handleDelete}
+        >
+          <Icon name="buttonDelete" />{' '}
+        </Button>
+      </Tooltip>
```

Comment:
What do you think about bringing this block to a separate component (nested inside this directory) and reusing it? Just a thought. Not mandatory.

Reply:
Fixed